internal/helpers/execute_commands.go

```diff
@@ -34,8 +34,20 @@ func Execute(commands []Command) error {
 			cmd.Stderr = os.Stderr
 		}
 
-		if err := cmd.Run(); err != nil {
-			return fmt.Errorf("cmd: can't execute command '%s %s'", c.Name, c.Options)
+		// Start execution of the command.
+		if err := cmd.Start(); err != nil {
+			return err
+		}
+
+		// Wait until this execution is complete.
+		if err := cmd.Wait(); err != nil {
+			// Define a new variable for the exit error.
+			var exitErr *exec.ExitError
+
+			// Check, if the current error is exit error.
+			if errors.As(err, &exitErr) {
+				return err
```

Comment:
sorry this cmd.Wait still return error cause
you generate package.json with an empty file instead of JSON format {}
![image](https://github.com/gowebly/gowebly/assets/24645546/826b0785-d3ba-4b9c-8c54-56dadcf5967a)
![image](https://github.com/gowebly/gowebly/assets/24645546/10486bbb-595a-45c1-b095-c02479b2b182)

so basically changes on file internal/helpers/execute_commands.go is to let you know why this error, but the root cause is because you created package.json with empty file instead of JSON format {}

Reply:
cc @koddr